Default (Linux).sublime-keymap

```diff
@@ -248,6 +248,11 @@ LaTeX Package keymap for Linux
 		"context":  [
 			{"key": "selector", "operator": "equal", "operand": "text.tex.latex"}],
 		"command": "latex_change_environment"},
+	// Toggle the starred version of environment
+	{ "keys": ["ctrl+l", "ctrl+shift+8"],
```

Comment:
Not sure if we should use this keybinding, because it is specific to the US layout.

Reply:
I'm not sure what a good alternative is.  I doesn't see like Sublime allows specifying `*` as a key.

Reply:
I also don't see a better alternative and it also works on other keyboards, so I think it is ok.